app/controllers/organizations_controller.rb

```diff
@@ -1,5 +1,5 @@
 class OrganizationsController < ApplicationController
-  before_action :find_organization, only: [:show, :edit, :update, :index]
+  before_action :find_organization, only: [:show, :edit, :update]
```

Comment:
rename "fetch" or "set" because find can be misleading vs. Active Records find and find_by method

Reply:
Renamed to fetch_organization